src/api/wallet/WalletApi.ts

```diff
@@ -279,10 +280,10 @@ export class WalletApiImpl implements WalletApi {
 
   /* ****************      Private Functions      **************** */
 
-  private async _notifyListeners(): Promise<void> {
-    await Promise.resolve()
+  private async _notifyListeners(options?: { blockHeader: { number: number } }): Promise<void> {
```

Comment:
who is using this options? The only other call to this private function doesn't provide any params

Reply:
Yeah, couldn't find a use for this. Guess @W3stside had some use in mind before but the part that I shamelessly copied has no use for it.

Will remove.

Reply:
Let's discuss it in person, cause I'm a bit confused by this